armadillo1/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,29 @@
+# Armadillo stack troubleshooting
+
+### "become" needs to work
+When you login to a VM you are hopefully yourself as in a useraccount that is recognisable as your account. After you logged in you need to be able to perform `sudo su` without entering a password. Get that in place and you will be able to run the playbook.
```

Comment:
When you login to a VM you are hopefully yourself as in a useraccount that is recognisable as your account

instead say

Login to the VM with the credentials you setup or are provided with.

Reply:
changed it with my own interpretation